src/mongocrypt-buffer.c

```diff
@@ -532,3 +532,11 @@ _mongocrypt_buffer_steal_from_string (_mongocrypt_buffer_t *buf, char *str)
    buf->owned = true;
    return true;
 }
+
+void
+_mongocrypt_buffer_copy_from_uint64 (_mongocrypt_buffer_t *buf, uint64_t value)
+{
+   _mongocrypt_buffer_init (buf);
+   _mongocrypt_buffer_resize (buf, 8);
+   memcpy (buf->data, &value, buf->len);
```

Comment:
~Okay, I take back my comment above. This appears to be machine dependent.  Is that your intent? I thought this was for representing a BSON int64 which needs to be fixed as LE. This will succeed on LE architectured, but fail on BE.~

Edit: I take back my take back, it looks like the macros below deal with endianness for us.  Still, at the minimum I'd like a comment reminding the reader that this particular function is NOT endian safe by itself and MUST be called with the endian conversion macros to be reliable across platforms.

Also, probably shouldn't hardcode the size at '8', but use `sizeof (value)`.

Reply:
Updated to use `sizeof (value)`.